lib/prefix.c

```diff
@@ -1090,6 +1167,22 @@ const char *prefix2str(union prefixconstptr pu, char *str, int size)
 		strlcpy(str, "FS prefix", size);
 		break;
 
+	case AF_LINKSTATE:
+		if (prefix_linkstate_display_hook)
+			snprintf(str, size, "%s/%d",
+				 prefix_linkstate_display_hook(
+					 buf, sizeof(buf),
```

Comment:
note for later: we should have an API to configure the buffer size length. (and perhaps an extensible length for the link state case).
today, it is hardset to PREFIX2STR_BUFFER.
if we increase it too much, we will have an impact on the memory size taken.
if we do not increase it enough, we will have a truncated prefix dump.

Reply:
OK